src/__tests__/index.js

```diff
@@ -40,10 +39,9 @@ test('can use pre-built glamorous components with css prop', () => {
   const computed = 'background'
   expect(
     render(
-      <glamorous.A
+      <glamorous.Span
```

Comment:
I've updated this to a `Span` because at some point, it was realized that `A` is a SVG element I guess ([tada](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/a)) which means that all valid SVG props can be forwarded to the `A` tag. This is actually quite unfortunate because we cannot actually tell ahead of time whether an `A` tag is used within an SVG or not. So when you use the `display` prop (as we did in this test), we cannot tell whether you want to forward it on or use it as CSS.

Today we forward it on rather than using it as CSS. It looks like this changed [11 days ago](https://github.com/jackyho112/react-html-attributes/commit/166ac11d40c7271a414b6c7862ca7f7afa7bf995) in [`react-html-attributes`](https://github.com/jackyho112/react-html-attributes), and it's causing the `display` prop to forward instead of be applied as a css value.

Let's deal with this in a separate issue, but I think we may want to reconsider some things around how we handle this...